source/Octopus.Data/Model/User/ActiveDirectoryIdentity.cs

```diff
@@ -0,0 +1,17 @@
+namespace Octopus.Data.Model.User
+{
+    public class ActiveDirectoryIdentity : ExternalIdentity
```

Comment:
There were 2 drivers. First was UI, currently the shape of the AD and OAuth identities has to be known because we can't do UI extensions yet. Investigating an idea on that front.

2nd, the Json serialization both on the wire and in the DB has direct knowledge atm. Working on a provider model for that so we can move the ActiveDirectoryIdentity and OAuthIdentity into the extensions.